File: docs/user/en/query/filtering/examples/comparable/attribute-between.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeBetween("battery-capacity", 125, 160)
				),
				require(
					entityFetch(
						attributeContent("code", "battery-capacity")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/comparable/attribute-between.cs
EvitaResponse<SealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeBetween("battery-capacity", 125, 160)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "battery-capacity")
        		)
        	)
        )
	)
);